src/Database/FunctionsBuilder.php

```diff
@@ -60,6 +60,16 @@ protected function _literalArgumentFunction($name, $expression, $types = [], $re
         return $this->_build($name, $expression, $types, $return);
     }
 
+    /**
+     * Returns a FunctionExpression representing a call to SQL RAND function.
+     *
+     * @return \Cake\Database\Expression\FunctionExpression
+     */
+    public function rand()
```

Comment:
Should the function name be `random()`?

Reply:
I'd stick to the short version to match other functions like "avg" and "sum".

Reply:
Fair enough.

Reply:
`rand()` comes from SQL `RAND()` function as in MySQL and SqlServer.